components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -55,6 +56,83 @@ fn json_unquote(arg: &Option<Json>) -> Result<Option<Bytes>> {
     })
 }
 
+fn json_with_bytes_validator(expr: &tipb::Expr) -> Result<()> {
```

Comment:
why is it called `json_with_bytes_validator` while it is used in `json_extract`?